src/snowflake/snowpark/mock/functions.py

```diff
@@ -69,7 +69,9 @@ def wrapper(*args, **kwargs):
 
 @patch("min")
 def mock_min(column: ColumnEmulator) -> ColumnEmulator:
```

Comment:
What window functions will we provide out of the box?

Reply:
I will refactor the current implementations (hardcoded `LEAD`, `LAG`, etc) into `functions.py` in a new PR.